### PR TITLE
bug: LT-1760 - Goods Total Value

### PR DESCRIPTION
**Pull request by andytaylorkainos**

### CODE REVIEW

File: templates/case/views/open-goods.html [hunk 1]
```diff
@@ -65,3 +65,5 @@ <h3 class="govuk-heading-m">Goods</h3>
 		{% endfor %}
 	</tbody>
 </table>
+<p class="govuk-body govuk-!-margin-top-3 govuk-!-margin-bottom-0 govuk-!-font-weight-bold">{% lcs 'STANDARD_CASE_TOTAL_VALUE' %} £{{ total_goods_value|intcomma }}</p>
+<br>
```
Comment (jfaracik) [inline templates/case/views/open-goods.html]: ACTUALLY. I lied. Don't use BR. Change `govuk-!-margin-bottom-0` to say `govuk-!-margin-bottom-3`

File: templates/case/views/standard-goods.html [hunk 2]
```diff
@@ -59,3 +59,4 @@
 	</tbody>
 </table>
 <p class="govuk-body govuk-!-margin-top-3 govuk-!-margin-bottom-0 govuk-!-font-weight-bold">{% lcs 'STANDARD_CASE_TOTAL_VALUE' %} £{{ total_goods_value|intcomma }}</p>
+<br>
```
Comment (jfaracik) [inline templates/case/views/standard-goods.html]: ACTUALLY. I lied. Don't use BR. Change `govuk-!-margin-bottom-0` to say `govuk-!-margin-bottom-3`

File: templates/case/views/open-goods.html (hunk 1 above)
Comment (andytaylorkainos) [reply templates/case/views/open-goods.html]: Done!

File: templates/case/views/standard-goods.html (hunk 2 above)
Comment (andytaylorkainos) [reply templates/case/views/standard-goods.html]: Done!

**Pull request merged**